k8_cortx_cloud/solution.yaml

```diff
@@ -39,6 +39,8 @@ solution:
       durability:
         sns: 1+0+0
         dix: 1+0+0
+    external_services:
+      type: LoadBalancer
```

Comment:
Do you expect this is a required field? If it's a temporary field then it's not required to be added to the "solution-check.yaml" file.

Reply:
It's something that we will want to expose at the highest level, even if the default is accepted. Right now, we don't have a way that simulates the functionality of the Helm values.yaml files that would allow us to have the user provide only the non-default values they want and everything else is defined in an internal values.yaml file in the chart. Since we are pulling all of our configuration information directly from the provided solution.yaml, I want to prevent building more logic into the bash scripts that will be going away in the future. The more logic we build into the solution.yaml and the associated charts, the better we'll be in the future.